ui-src/src/app/test-cases/test-cases.component.ts

```diff
@@ -33,6 +33,7 @@ export class TestCasesComponent implements OnInit {
   }
 
   onLoad() {
+    console.log(this.testCasePath);
```

Comment:
if you log this, put in a string so we know what to look for in the console
e.g. console.log("Test case path: "+this.testCasepath)